README.md

```diff
@@ -56,6 +56,23 @@ cd immutable-re
 opam pin add -y immutable .
 ```
 
+## Releasing a new version
```

Comment:
Can you this in another file. Its only really applicable to maintainers. Perhaps RELEASE.txt, etc.